tests/integration/test_bias_metrics.py

```diff
@@ -163,11 +163,12 @@ def test_bias_metrics():
                     "value": pytest.approx(0.06494661921708189),
                 },
                 {"name": "FT", "description": "Flip Test (FT)", "value": pytest.approx(-0.32373271889400923)},
+                {"name": "GE", "description": "Generalized Entropy (GE)", "value": 0.04922014234295094},
```

Comment:
Wrap the number by pytest.approx(), the same comment to the other new metric value.

Reason: avoid precision issue of comparing floating number directly.

Reply:
missed this thanks. Done. Yes I've ran `./devtool all` and `./devtool integ_tests` so the fix works and tests are passing. This test isn't included in `./devtool all`.